src/language/providers/intellisense/attributeValueItems.ts

```diff
@@ -0,0 +1,294 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// prettier-ignore
```

Comment:
What needs ignoring?

Reply:
Initially the file started out as a copy of attributeItems.ts.  But the suggestions first offered a popup of the attribute name, once the attribute name was selected, then the choice values were given. I found that by using a 'switch" statement I was able to supply the attribute choices directly without the user having to select the attribute name first. When I made the change I forgot the prettier-ignore directive was in the file.  I removed it.